composer.json

```diff
@@ -11,16 +11,16 @@
     "require": {
         "php": "^8.0",
         "ext-json": "*",
-        "doctrine/doctrine-module": "^4.4.0",
-        "doctrine/doctrine-orm-module": "^4.2.1",
-        "laminas/laminas-cache": "^3.1.3",
         "laminas/laminas-db": "^2.9",
         "laminas/laminas-dependency-plugin": "^2",
         "laminas/laminas-eventmanager": "^3.2",
         "laminas/laminas-http": "^2.8",
         "laminas/laminas-log": "^2.10",
         "laminas/laminas-servicemanager": "^3.10",
-        "laminas/laminas-stdlib": "^3.2"
+        "laminas/laminas-stdlib": "^3.2",
```

Comment:
I'd probably keep these in the order they were in, just more readable alphabetical